docs/unified-test-documentation/test-matrix.md

```diff
@@ -0,0 +1,148 @@
+# Test matrix
```

Comment:
I wonder if below approach is correct. It was like that in the past, but it seem that from sales perspective there is more interest in division already presented [here](https://github.com/Dasharo/docs/pull/54#issuecomment-999508964)